apps/prairielearn/src/tests/access.test.ts

```diff
@@ -46,39 +46,39 @@ describe('Access control', { timeout: 20000 }, function () {
       2450 after course_instance
      */
 
-  function cookiesStudent() {
+  async function cookiesStudent() {
     const cookies = new fetchCookie.toughCookie.CookieJar();
-    cookies.setCookie('pl_test_user=test_student', siteUrl);
+    await cookies.setCookie('pl_test_user=test_student', siteUrl);
```

Comment:
There's a `setCookieSync(...)` function, IMO it'd be better to use that? I don't see a reason that this needs to be async.

Reply:
Fixed in 6d3ab0659925d5cb29dd02ca81164ee829466c43.